tests/perf/pubsub_subscribe_http/test.js

```diff
@@ -11,24 +11,24 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-import http from 'k6/http'
-import { check } from 'k6'
-import exec from 'k6/execution'
-import ws from 'k6/ws'
-import { Counter } from 'k6/metrics'
+import http from "k6/http";
```

Comment:
Please run `make me prettier` so this is formatted consistently

(you will need Node.js installed, and you may need to do `make prettier-install` first)